Detector/DetDriftChamber/src/driftchamber/DriftChamber.cpp

```diff
@@ -210,7 +213,7 @@ static dd4hep::Ref_t create_detector(dd4hep::Detector& theDetector,
             dd4hep::PlacedVolume module_phy = layer_vol.placeVolume(module_vol,transform_module);
            // - Field wire
             dd4hep::PlacedVolume Module_phy;
-            double radius[9] = {rmid-chamber_layer_width*0.5,rmid-chamber_layer_width*0.5,rmid-chamber_layer_width*0.5,rmid-chamber_layer_width*0.5,rmid,rmid+chamber_layer_width*0.5,rmid+chamber_layer_width*0.5,rmid+chamber_layer_width*0.5,rmid+chamber_layer_width*0.5};
+            double radius[9] = {rmid-chamber_layer_width*0.5+safe_diatance,rmid-chamber_layer_width*0.5+safe_diatance,rmid-chamber_layer_width*0.5+safe_diatance,rmid-chamber_layer_width*0.5+safe_diatance,rmid,rmid+chamber_layer_width*0.5-safe_diatance,rmid+chamber_layer_width*0.5-safe_diatance,rmid+chamber_layer_width*0.5-safe_diatance,rmid+chamber_layer_width*0.5-safe_diatance};
```

Comment:
Could you explain more about this line?

Reply:
Ok. Because the field wire was placed in the middle of the two layers before, it would cause overlap, so I increased the position of the field wire outward by a safe value, among which, for the field wire on the outer side of the outermost layer of the two drift chambers , moved inward by a safe value.

Reply:
Your explanation here is not very clear to me. Could you add a schematic diagram to explain it? Thanks.
Tao

Reply:
Old version：the field wire was placed in the middle of the two layers，as shown，- - - -O- - - -，  O means field wire.
New version: The position of the field wire inside the layer，_ _ _ _O_ _ _ _ ,   O means field wire.